backend/handlers/handlers.go

```diff
@@ -1543,10 +1559,16 @@ func (handlers *Handlers) getLightningConfigHandler(_ *http.Request) interface{}
 	lightningAccounts := handlers.backend.Config().LightningConfig().Accounts
 	accounts := make([]*lightningAccountConfigWithoutMnemonic, len(lightningAccounts))
 	for i, account := range lightningAccounts {
+		keystoreName := ""
+		lightningKeystore, err := handlers.backend.Config().AccountsConfig().LookupKeystore(account.RootFingerprint)
+		if err == nil {
+			keystoreName = lightningKeystore.Name
```

Comment:
This is an interesting solution, but I think it has a defect: if the keystore device is renamed, the config in the frontend is not updated bc it is served by the provider. That means the keystore name won't be updated until the app is restarted. I think it would be better to add a backend endpoint `keystore-name` that given a rootfingerprint looks for a keystore in the config and returns the name. 

There is also a more latent bug, which depends on the watch-only implementation: when the device is renamed, the keystore name in the config file is not updated until the device is disconnected and reconnected again. But this deserves its own PR as it is out of scope here, imo. 

cc @benma